src/backend/api/Fusion.Resources.Api/Controllers/Person/ApiModels/ApiPersonAbsence.cs

```diff
@@ -26,10 +26,31 @@ public ApiPersonAbsence(QueryPersonAbsence absence, bool hidePrivateNotes)
                 TaskDetails = (absence.TaskDetails != null) ? ApiTaskDetails.Hidden : null;
             }
         }
+        public ApiPersonAbsence(QueryPersonAbsenceBasic absence, bool hidePrivateNotes)
+        {
+            Id = absence.Id;
+            AppliesFrom = absence.AppliesFrom;
+            AppliesTo = absence.AppliesTo;
+            Type = Enum.Parse<ApiAbsenceType>($"{absence.Type}", true);
+            AbsencePercentage = absence.AbsencePercentage;
+
+            IsPrivate = absence.IsPrivate;
+            Comment = absence.Comment;
+            TaskDetails = (absence.TaskDetails != null) ? new ApiTaskDetails(absence.TaskDetails) : null;
+
+            if (hidePrivateNotes && absence.IsPrivate)
+            {
+                Comment = "Not disclosed.";
```

Comment:
What happens if a user with limited access issues a GET and a PUT (if that scenario is possible) ?
Not sure if it is appropriate to modify updateable properties.

Reply:
The permissions required to update prevents this from happening.